crm_phonecall_summary_predefined/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,5 @@
+To configure the possible summary options:
+
+#. Activate the developer mode
+#. Go to *CRM > Configuration > Leads & Opportunities > Phone Calls > Summaries*.
```

Comment:
New menu path should be CRM => Configuration => Pipeline => Phone Calls => Summaries

Reply:
Done